src/main/groovy/org/shipkit/internal/gradle/downstream/test/RunDownstreamTestTask.java

```diff
@@ -15,9 +15,9 @@
 /**
  * This task run external process and additionally store output of external process to file.
  */
-public class RunTestReleaseTask extends DefaultTask {
+public class RunDownstreamTestTask extends DefaultTask {
 
-    private static final Logger LOG = Logging.getLogger(RunTestReleaseTask.class);
+    private static final Logger LOG = Logging.getLogger(RunDownstreamTestTask.class);
 
```

Comment:
The 'repoName' property on this task is somewhat awkward. It's only used for logging the message. Perhaps we can push the message to doFirst action? Or change the 'repoName' property to 'displayName'. Then, in code we would print

```Java
LOG.lifecycle("  Running {}. The output will be saved in {}", ...); 
```